app/forms/views/submission.py

```diff
@@ -57,6 +60,12 @@ def create(self, request, *args, **kwargs):
                     status=status.HTTP_400_BAD_REQUEST,
                 )
 
+            if not form.event.registrations.filter(user=user).exists():
+                allow_overwrite = True
+
+        if allow_overwrite:
+            Submission.objects.filter(user=user, form=form).delete()
```

Comment:
Hva med denne?
```suggestion
            if not form.event.registrations.filter(user=user).exists():
                return super().update(request, *args, **kwargs)
```
Kan også være en idé å flytte det til modellen så vi holder logikken på et så lavt nivå som mulig

Reply:
Ja, jeg tenkte det samme. Eneste problemet (hvis jeg husker rett) er at submission ikke vet om det tilhører et event eller ikke. Jeg mener at man får baseclassen om man bruker `form` i modellen, men jeg skal dobbelsjekke. :wink: